snap/snapcraft.yaml

```diff
@@ -0,0 +1,56 @@
+name: chisel
+summary: Chisel is a software tool for carving and cutting Debian packages!
+description: |
+  Chisel can slice a Linux distribution using a release database
+  and construct a new filesystem using the finely defined parts.
+
+  Usage: chisel <command> [<options>...]
+
+  Commands can be classified as follows:
+
+    Basic: help, version
+    Action: cut
+
+  For more information about a command, run 'chisel help <command>'.
+  For a short summary of all commands, run 'chisel help --all'.
```

Comment:
Additionally, you should also describe the snap's MO, mentioning that users can only `cut` slices into the $HOME directory